.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!--
+Please have a look at the issue templates you get when you click "New issue" in the GitHub UI.
+We very much prefer issues created by using one of these templates.
+-->
```

Comment:
How about

> Did you have a look at the issue templates you get when you click "New issue" in the GitHub UI?
Please only create an issue here if it absolutely does not fit into one of these categories.

Furthermore, I'd ditch the rest of this template. Since we don't know exactly what the issue is about, we should not make any assumptions. The template still assumes some kind of problem, however.

The one use-case for this I can think of right now is developer tasks. Like "Drop support for Node.js 4". I'd prefer a blank template for these.

Reply:
This template was my try to reinforce to users who use it, that this should normally be about bugs - and not questions ;)

As all users can just delete whatever is in the text field (and many do...) I didn't think it was too confusing. Wrong?

Reply:
Well, I was mostly hoping not having to start every issue with `Ctrl+A, Del` anymore. :sweat_smile: